src/test/java/jp/pay/PayjpTest.java

```diff
@@ -180,6 +180,12 @@ public void testAPIException() throws PayjpException {
 		Charge.create(defaultChargeParams);
 	}
 
+	@Test(expected=APIException.class)
+	public void testAPIExceptionWithHtmlResponse() throws PayjpException {
```

Comment:
できる限り子クラスの方がテストとしては良さそう
```suggestion
	public void testAPIExceptionWithHtmlResponse() throws APIException {
```